conf/supervisord-dev.conf

```diff
@@ -3,7 +3,7 @@ nodaemon = true
 silent = true
 
 [program:web]
-command=gunicorn --paste conf/development.ini
+command=gunicorn --workers 1 --paste conf/development.ini
```

Comment:
Made the default explicit so it's easier to change.

Reply:
You _should_ be able to set this in the `conf/development.ini`, but I have a recollection that certain fields like the port and stuff just don't work:

 * https://docs.gunicorn.org/en/stable/configure.html#paster-applications
 
Honestly thought I'm not sure if that's just more confusing than having it here

Reply:
I tried a few of those sections in the .ini and didn't work. At the end I was happy with anywhere I could grep for `workers`.